cpp/include/cudf/io/text/multibyte_split.hpp

```diff
@@ -87,18 +88,29 @@ std::unique_ptr<cudf::column> multibyte_split(
   data_chunk_source const& source,
   std::string const& delimiter,
   parse_options options             = {},
+  rmm::cuda_stream_view stream      = cudf::get_default_stream(),
   rmm::device_async_resource_ref mr = rmm::mr::get_current_device_resource());
 
-std::unique_ptr<cudf::column> multibyte_split(
```

Comment:
is this missing documentation not flagged by doxygen pre-commit check?

Reply:
You are right. Looks like this API has not appeared in our doxygen thus far.
I'll add it to the `io_readers` group.